core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,39 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
```

Comment:
Exploration-player-specific stuff should probably go in the exploration player file.

That said, the stuff in the "max-width:768px" media rule seems generic. Can you confirm that you have carefully checked all the other pages and are happy with how things look there? Note that some other pages, like the exploration editor, have a custom navbar, so you need to be careful and thorough to ensure that there are no regressions.